settlement/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
Please confirm the purpose of this service, and how it differs from the central-settlement service?